.travis.yml

```
@@ -0,0 +1,12 @@
+language: python
```

Please also use the latest Ubuntu LTS
```suggestion
dist: bionic
language: python
```